recipes/geocat-viz/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "geocat-viz" %}
+{% set version = "2021.08.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NCAR/{{ name }}/archive/v{{ version }}.tar.gz   # For building from a specific tag's tarball
+  sha256: 6624ffb855cbe717a8eb624c7fa313a47bf1b3827e22f28f4dec8b2e298d47cd
+  # git_tag: v{{ version }}   # For building from a specific tag
+  # git_rev: main # For building from main branch for debugging purposes
+  # git_url: https://github.com/NCAR/{{ name }}.git
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed -vv .
+
+requirements:
+  build:
+    - python                              # [build_platform != target_platform]
+    - pip                                 # [build_platform != target_platform]
+    - setuptools                          # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+  host:
+    - python >=3.8
+    - pip
+    - setuptools
+  run:
+    - python >=3.8
+    - cartopy
+    - cmaps
+    - numpy
+    - scikit-learn
+    - xarray
+
+test:
+  requires:
+    - pip
+    - cmaps
+    - numpy
+    - matplotlib
+    - scikit-learn
+  imports:
+    - geocat
+    - geocat.viz
+    - geocat.viz.util
+  commands:
+    - pip check
+  source_files:
+    - src
+
+about:
+  home: https://geocat.ucar.edu/
+  summary: GeoCAT-viz is a collection of utility functions to make Python plotting with Matplotlib and Cartopy more like NCL (NCAR Command Language).
+  dev_url: https://github.com/NCAR/{{ name }}
```

Comment:
The use of the variable name here makes the recipe hard to read and links impossible to follow. Also, having a name var vs the actual name here does means you plan to change the name, which we hope you do not :smile:

TL;DR we only use jinja variables where they either help the bot or with thing we will change with every updating PR, like the version.

Reply:
k, if you want a hardcoded name, we'll hardcode the name. 
we don't plan to change it.

Reply:
resolved